etc/init.d/docker-compose

```diff
@@ -8,10 +8,28 @@ depend() {
 
 start() {
   ebegin "Starting docker-compose"
-
-  cd /home/lncm/compose
-  docker-compose up -d
-
+  if [[ -d /media/important/important/lnd ]]; then
+    if [[ -d /media/archive/archive/bitcoin ]]; then
+      if [[ -f /media/important/important/lnd/lnd.conf ]]; then
+        if [[ -f /media/archive/archive/bitcoin/bitcoin.conf ]]; then
```

Comment:
the `if` statements would be easier to deal with when they are not nested

Reply:
what linting errors

Reply:
I just pushed || exit to the lines